### PR TITLE
Release 2.6.1

### PR DESCRIPTION
**Pull request by F4ever**

### CODE REVIEW

**Review by avsetsin — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [comment]: <> (## [Unreleased]&#40;https://github.com/lidofinance/lido-oracle&#41; - 2021-09-15)
 
+## [2.6.1](https://github.com/lidofinance/lido-oracle/releases/tag/2.6.1) - 2023-04-08
+### Fixed
+- Correctly handle missed slot.
```
Comment (TheDZhon) [inline CHANGELOG.md:12]: I would rather been more specific

File: app/beacon.py [hunk 2]
```diff
@@ -58,6 +58,14 @@ def inner(*args, **kwargs):
     return inner
 
 
+class BeaconBlockNotFoundError(Exception):
+    pass
+
+
+class NoNonMissedSlotsFoundException(Exception):
```
Comment (TheDZhon) [inline app/beacon.py:65]: NoSlotsForExpectedEpoch or smth?

File: app/metrics.py [hunk 3]
```diff
@@ -70,8 +70,7 @@ def get_full_current_metrics(
     w3: Web3, pool, beacon, beacon_spec, partial_metrics, consider_withdrawals_from_epoch
 ) -> PoolMetrics:
     """The oracle fetches all the required states from ETH1 and ETH2 (validator balances)"""
-    slots_per_epoch = beacon_spec[1]
-    slot = partial_metrics.epoch * slots_per_epoch
+    slot = beacon.get_slot_for_report(partial_metrics.epoch * beacon_spec[1], beacon_spec[0], beacon_spec[1])
```
Comment (TheDZhon) [inline app/metrics.py:73]: maybe should have the names exposed instead of raw indexes

**Review by TheDZhon — COMMENTED** [review]
LGTM :+1:

**Pull request merged**